line_obfuscator.go

```diff
@@ -128,11 +124,14 @@ func (tf *transformer) transformLineInfo(file *ast.File, name string) (detachedC
 		case *ast.GenDecl:
 			doc = &decl.Doc
 		}
-		newPos := fmt.Sprintf("%s%c.go:%d",
-			prefix,
-			nameCharset[mathrand.Intn(len(nameCharset))],
-			PosMin+newLines[i],
-		)
+		newPos := prefix + ":1"
```

Comment:
Didn't just `//line :1` also work for tiny? (Why the prefix)

Reply:
because cgo will complain if the filename doesn't have the cgo prefix. the cgo test starts failing if this is omitted